client/src/components/Menu.jsx

```diff
@@ -53,14 +70,15 @@ class Menu extends React.Component {
           <h1 className="menu-title">Menu</h1>
           <div className="menuBtnContainer">
             {this.subMenusList.map((subMenu, i) => {
-              return <MenuButton name={subMenu} selected={this.state.selected} key={i} 
+              return <MenuButton name={subMenu} selectedSubMenu={this.state.selectedSubMenu} key={i} 
                       handleClick={this.handleMenuBtnClick}/>;
```

Comment:
add a space after the {this.handleMenuBtnClick}

Reply:
Also probably just a preference thing, but I saw someone else put the /> on a new line of it's own. But solely pref I think might be bad since it takes up more space.